sdk/sdk-view/src/test/java/mil/emp3/api/MirrorCacheTest.java

```diff
@@ -17,11 +22,11 @@
 import static org.mockito.Mockito.when;
 import static org.powermock.api.mockito.PowerMockito.mock;
 */
-
+@RunWith(RobolectricTestRunner.class)
 public class MirrorCacheTest extends TestCase {
 
     // Keep JUnit happy, throw it a bone
-
+    @Test
```

Comment:
Should probably just comment this entire file out if its not actually testing anything